python/adelphi/adelphi/anonymize.py

```diff
@@ -0,0 +1,96 @@
+# Functions and constants related to the anonymization process
+
+from adelphi.store import get_standard_columns_from_table_metadata
+
+# default prefixes for the anonymized names
+KEYSPACE_PREFIX = "ks"
+TABLE_PREFIX = "tbl"
+PARTITION_KEY_PREFIX = "pk"
+CLUSTERING_KEY_PREFIX = "ck"
+COLUMN_PREFIX = "col"
+TYPE_PREFIX = "udt"
+FIELD_PREFIX = "fld"
+INDEX_PREFIX = "idx"
+
+# maps the original schema names to the replacement names
+name_map = {
+    KEYSPACE_PREFIX: {},
+    TABLE_PREFIX: {},
+    PARTITION_KEY_PREFIX: {},
+    CLUSTERING_KEY_PREFIX: {},
+    COLUMN_PREFIX: {},
+    TYPE_PREFIX: {},
+    FIELD_PREFIX: {},
+    INDEX_PREFIX: {}
+}
```

Comment:
Might need to re-think this: if we run multiple schemas through the anonymizer this probably should become a per-schema map which means we'd need to regenerate it on a per-keyspace basis.  It works as-is but the isolation offered by moving this to a per-keyspace thing might be (slightly) better.

Reply:
Does it matter to have different anonymized names per schema? I think as long it's anonymous and consistent across the generated CQL statements, we don't need to worry about re-using a name.

Reply:
Yeah, that's what I meant by "works as-is" above.  I'm not suggesting it's broken, I'm just wondering if it isn't more straightforward to manage on a per-keyspace basis.  Certainly not essential, just something I was musing about.